dfm/cleaning/deduper.py

```diff
@@ -116,14 +137,35 @@ def _get_minhash(self, doc: str) -> LeanMinHash:
             ]
         elif self.split_method == "paragraph":
             shingles = [p for p in doc.split("\n") if len(p) > 0]
-        elif self.split_method == "none":
+        elif self.split_method == "none" or self.split_method is None:
```

Comment:
We might wanna remove the option to give a "none" string?

Reply:
It's quite convenient though, and solves confusion. Say, if I want to create a simple CLI, then input will always be a string, and that enables me to simply write 'none'.

Reply:
Hmm, I would probably argue for removing the NoneType as an option, which would make sure that the split_method is always a string.